src/app/contacts/main-contacts.component.css

```diff
@@ -0,0 +1,15 @@
+.main-container {
+  margin: 30px 80px;
+}
+.card {
```

Comment:
Sicer stvar okusa ampak pri nas ponavadi za padding in margine uporabljamo predefinirane boostrap classe.

Reply:
Se strinjam, takšen pristop pripomore k enotnejšemu pozicioniranju elementov na formi in je s tem manj prostora za napake/težave.